uaclient/config.py

```diff
@@ -737,6 +785,26 @@ def process_config(self):
                 )
             )
 
+    def write_cfg(self, config_path=None):
+        """Write config values back to config_path or DEFAULT_CONFIG_FILE."""
+        if not config_path:
+            config_path = DEFAULT_CONFIG_FILE
+        content = status.MESSAGE_UACLIENT_CONF_HEADER
+        cfg_dict = copy.deepcopy(self.cfg)
+        if "log_level" not in cfg_dict:
+            cfg_dict = CONFIG_DEFAULTS["log_level"]
```

Comment:
@blackboxsw I think this is supposed to set the "log_level" key value
```suggestion
            cfg_dict["log_level"] = CONFIG_DEFAULTS["log_level"]
```

Reply:
+1 thanks